cmds/modules/provisiond/swagger/zos-api.yml

```diff
@@ -468,6 +470,28 @@ components:
           type: string
         disable_default_ingress:
           type: boolean
+    VirtualMachine:
```

Comment:
in swagger, you can also do some sort of "inheritance" so a K8s struct can include the attrs from virtualmachine. look it up